tools/UpdatePaths/replaceplasteelturf.txt

```diff
@@ -46,5 +46,3 @@
 /turf/open/floor/plasteel/white/corner : /turf/open/floor/iron/white/corner {@OLD}
 /turf/open/floor/plasteel/white/side : /turf/open/floor/iron/white/side {@OLD}
 /turf/open/floor/plasteel/white/telecomms : /turf/open/floor/iron/white/telecomms {@OLD}
-/turf/open/floor/plasteel/yellowsiding : /turf/open/floor/iron/yellowsiding {@OLD}
-/turf/open/floor/plasteel/yellowsiding/corner : /turf/open/floor/iron/yellowsiding/corner {@OLD}
```

Comment:
you should put this in the turfs2decals.txt script and make it convert those tiles into the siding decals

Reply:
Sounds good! And I can change those back to the lines if you like. 👍